client/web/src/user/settings/research/ProductResearch.tsx

```diff
@@ -33,6 +35,38 @@ export const ProductResearchPage: React.FunctionComponent<React.PropsWithChildre
         <>
             <PageTitle title="Product research" />
             <PageHeader headingElement="h2" path={[{ text: 'Product research and feedback' }]} className="mb-3" />
+            {isSourcegraphApp && (
+                <Container className="mb-2">
+                    <Text>Do you have feedback or need help with Sourcegraph App?</Text>
+                    {[
+                        {
+                            content: 'File an issue',
+                            path: 'https://github.com/sourcegraph/app',
+                            variant: 'primary' as const,
+                        },
+                        {
+                            content: 'Join our Discord',
+                            path: 'https://about.sourcegraph.com/community',
+                        },
+                        {
+                            content: 'Follow us on Twitter',
+                            path: 'https://twitter.com/sourcegraph',
+                        },
+                        { content: 'Email us', path: 'mailto:taylor.sperry@sourcegraph.com' },
```

Comment:
```suggestion
                        { content: 'Email us', path: 'mailto:support@sourcegraph.com' },
```

lol probably best this goes to a catch-all; either that or we can set up a dedicated email support group for it.

Reply:
Haha yeah this was part of this issue: https://github.com/sourcegraph/sourcegraph/issues/48397

Although I’m actually removing it now. There's some discussion in Figma that too many options are confusing and I'd rather have this in sync with the header: https://github.com/sourcegraph/sourcegraph/pull/48838